packages/exemplar-sample/package.json

```diff
@@ -28,10 +28,12 @@
   "homepage": "https://github.com/godaddy/exemplar/tree/master/packages/exemplar-sample",
   "dependencies": {
     "@exemplar/storybook-react": "^1.1.0",
-    "@exemplar/storybook-react-native": "^1.0.1",
+    "@exemplar/storybook-react-native": "^1.0.0",
     "@storybook/addon-knobs": "^5.1.9",
     "@storybook/addons": "^5.1.9",
+    "@storybook/core": "^5.1.11",
```

Comment:
Probably best to keep this version in sync with the other storybooks. I think they version everything at the same time. So either make this ^5.1.9 or the others ^5.1.11 to match this one.

Reply:
👍